native/machine-coding/Weather App/style.css

```diff
@@ -0,0 +1,111 @@
+@import url(../../helpers/core.css);
+
+*{
+    margin: 0;
+    padding: 0;
+    font-family: 'Poppins', sans-serif;
+    box-sizing: border-box;
+}

```

Comment:
above css ruleset is not needed as we import core

Reply:
okay